roles/network_plugin/multus/tasks/main.yml

```diff
@@ -0,0 +1,50 @@
+---
+- name: Multus | Delete existing configuration
+  file:
+    state: absent
+    path: "{{ multus_conf_dir }}/{{ multus_conf_filename }}"
+
+- name: Multus | Read CNI configuration of master plugin
+  shell: |
+      cd {{ multus_conf_dir }}
+      cat $(ls -p | grep -v / | head -1)
+  register: output
+
+- name: Multus | Save CNI configuration of master plugin
+  set_fact:
+    masterplugin_conflist: "{{ output.stdout | from_json | to_nice_json }}"
+
+- name: Multus | Generate unformatted Multus CNI configuration file
+  template:
+    src: "cni-multus.conf.j2"
+    dest: "{{ multus_conf_dir }}/{{ multus_conf_filename }}"
+    owner: kube
+
+- name: Multus | Load Multus CNI configuration for formatting
+  command: "cat {{ multus_conf_dir }}/{{ multus_conf_filename }}"
+  register: output
+
+- name: Multus | Store formatted Multus CNI configuration
+  copy:
+    content: "{{ output.stdout | from_json | to_nice_json }}"
+    dest: "{{ multus_conf_dir }}/{{ multus_conf_filename }}"
+    owner: kube
+
+- name: Multus | Copy manifest files
+  copy:
+    src: "{{ item.file }}"
+    dest: "{{ kube_config_dir }}"
+  with_items:
+    - {name: multus-crd, file: multus-crd.yml, type: customresourcedefinition}
+    - {name: multus-serviceaccount, file: multus-serviceaccount.yml, type: serviceaccount}
+    - {name: multus-clusterrole, file: multus-clusterrole.yml, type: clusterrole}
+    - {name: multus-clusterrolebinding, file: multus-clusterrolebinding.yml, type: clusterrolebinding}
+  register: multus_manifest_1
+
+- name: Multus | Copy manifest templates
+  template:
+    src: "{{ item.file }}.j2"
+    dest: "{{ kube_config_dir }}/{{ item.file }}"
+  with_items:
+    - {name: multus-daemonset, file: multus-daemonset.yml, type: daemonset}
```

Comment:
Is this file generated or does it need to be added?

Reply:
Good catch. I missed adding the file to the PR. It's there now.